### PR TITLE
Sign up mobile

### PR DESCRIPTION
**Pull request by Ervige**

### CODE REVIEW

File: pages/housePrideQuiz/signUpMobile.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="../../index.css" rel="stylesheet" />
+    <link href="signUpMobile.css" rel="stylesheet" />
+    <title>Register</title>
+  </head>
+    <header>
+        <main>
+          <div class="menu">
+            <figure>
+            <a href="/pages/housePrideQuiz/celebrateHousePride.html"><img
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/signUpMobile.html:15]: Please add ENTER after <a> tag. Please remember about proper code format.

**Review by AlpacaInTheCode — CHANGES_REQUESTED** [review]
Please fix all of the issues.

File: pages/housePrideQuiz/signUpMobile.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            <img 
+            src="https://www.wizardingworld.com/images/icons/diamonds.svg"
+            alt="Three squares"
+            class="sign__picture"
+            />
+            <hr class="sign__line">
+        </div>
+        <form class="form">
+            <div class="form__date">
+            <label for="form__birth">DATE OF BIRTH</label>
```
[the remark this answers is not in the document]
Comment (Ervige) [reply pages/housePrideQuiz/signUpMobile.html]: @AlpacaInTheCode fixed

File: pages/housePrideQuiz/signUpMobile.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                <input class="form__placeholder" type="text" placeholder="DD/MM/YYYY" name="date" maxlength="10" required />
+            </div>
+        <div class="form__email">
+            <label for="form_mail">EMAIL ADDRESS</label>
+                <input class="form__placeholder" type="text" placeholder="Enter Email" name="email" required />
+        </div>
+        <div class="form__password">
+            <label for="form_pass">PASSWORD</label>
+                <input class="form__placeholder" type="password" name="password" minlength="8" autocapitalize="none" autocomplete="off"  required />
```
[the remark this answers is not in the document]
Comment (Ervige) [reply pages/housePrideQuiz/signUpMobile.html]: @AlpacaInTheCode fixed

File: pages/housePrideQuiz/signUpMobile.html (hunk 1 above)
Comment (Ervige) [reply pages/housePrideQuiz/signUpMobile.html]: @AlpacaInTheCode fixed

**Review by AlpacaInTheCode — APPROVED** [review]

**Pull request merged**